assets/js/components/notifications/WPVersionBumpNotification.js

```diff
@@ -0,0 +1,98 @@
+/**
+ * WordPress Version Upgrade Notification component.
+ *
+ * Site Kit by Google, Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import { useMount } from 'react-use';
+
+/**
+ * WordPress dependencies
+ */
+import { __, sprintf } from '@wordpress/i18n';
+import { useCallback } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import Data from 'googlesitekit-data';
+import { CORE_SITE } from '../../googlesitekit/datastore/site/constants';
+import BannerNotification from './BannerNotification';
+import { getTimeInSeconds, trackEvent } from '../../util';
+import useViewContext from '../../hooks/useViewContext';
+const { useSelect } = Data;
+
+export default function WPVersionBumpNotification() {
+	const viewContext = useViewContext();
+	const eventCategory = `${ viewContext }_wp52-version-notification`;
+
+	const handleOnDismiss = useCallback( () => {
+		trackEvent( eventCategory, 'dismiss_notification' );
+	}, [ eventCategory ] );
+
+	const handleOnCTAClick = useCallback( () => {
+		trackEvent( eventCategory, 'confirm_notification' );
+	}, [ eventCategory ] );
+
+	useMount( () => {
+		trackEvent( eventCategory, 'view_notification' );
+	} );
+
+	const hasMinimumWPVersion = useSelect( ( select ) =>
+		select( CORE_SITE ).hasMinimumWordPressVersion()
```

Comment:
Ah, this needed to be updated but wasn't, so it's failing all of the E2E tests. Here's the fix:

```suggestion
		select( CORE_SITE ).hasMinimumWordPressVersion( '5.2' )
```

Reply:
My bad, I missed it. Thanks for fixing it, @tofumatt 👍